chips/nrf52/src/uart.rs

```diff
@@ -121,6 +120,10 @@ impl UART {
         let regs = unsafe { &*self.regs };
         let tx = regs.event_endtx.get() != 0;
 
+        if regs.event_error.get() == 1 {
+            panic!("uart error event\r\n");
```

Comment:
It _might_ be reasonable to not do error handling, depending on what an error could mean